Core/Libraries/Source/Compression/Compression.h

```diff
@@ -25,7 +25,7 @@
 #ifndef __COMPRESSION_H__
 #define __COMPRESSION_H__
 
-#include "Lib/BaseType.h"
+#include "Lib/BaseTypeCore.h"
```

Comment:
How about put core basetype.h into Core/Lib/BaseType.h that is visible to the games like that, but is visible as "Lib/BaseType.h" to Core, so that no header needs changing?

Reply:
That is true. But then it means all `#include "Lib/BaseType.h"`'s will need to be renamed on moving files.

Alternatively can put a `Core/Libraries/Adapter/Lib/BaseType.h`, that just includes `"Lib/BaseTypeCore.h"`. Generals and GeneralsMD will not see `Core/Libraries/Adapter`, so there is no ambiguity. Then it will not be necessary to rename any `#include "Lib/BaseType.h"`.

Reply:
I prefer to not rename headers when technically not necessary. And here it is not necessary. I can push a change to this branch if you are unsure how to do it.